packages/strapi-admin/controllers/Auth.js

```diff
@@ -321,21 +327,50 @@ module.exports = {
     // Generate random token.
     const resetPasswordToken = crypto.randomBytes(64).toString('hex');
 
-    const settings = {
-      from: {
-        name: 'Administration Panel',
-        email: 'no-reply@strapi.io',
-      },
-      response_email: '',
-      object: 'Reset password',
-      message: `<p>We heard that you lost your password. Sorry about that!</p>
+    // Retrieve email settings from database. Enable the use of external email plugins.
```

Comment:
Can you update this comment and remove the second sentence, please.
This not impact the usage of email plugin itself.

Reply:
Done 😄